config/config-tree.json

```diff
@@ -0,0 +1,1968 @@
+{
```

Comment:
we want to generate data source for UI outside of config folder to make sure merging schema stays without any issue. Can you mark the target for the output of the script to be outside config in another place?